pkg/controller/podautoscaler/horizontal.go

```diff
@@ -173,8 +173,9 @@ func (a *HorizontalController) Run(stopCh <-chan struct{}) {
 		return
 	}
 
-	// start a single worker (we may wish to start more in the future)
-	go wait.Until(a.worker, time.Second, stopCh)
+	for i := 0; i < workers; i++ {
```

Comment:
Workers num should has an limit, in case for too much goroutine.

Reply:
There is not a proper value we can limit the number of workers, just describing in flag usage